bundle/render_templates.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -x
+
+env
+
+csv_name="cert-manager-operator.clusterserviceversion.yaml"
+csv_file="${MANIFESTS_DIR}/${csv_name}"
+if [ ! -f $csv_file ]; then
+   echo "CSV file not found in the upstream!"
+   exit 5
+fi
+
+yq e -i ".metadata.annotations.createdAt=\"$(date -u +'%Y-%m-%dT%H:%M:%S')\"" "${csv_file}"
+yq -i 'del(.spec.relatedImages)' "${csv_file}"
```

Comment:
We need to ensure some pipeline adds back the relatedImages after they've been deleted here. The final bundle ideally needs to contain `spec.relatedImages` with exact container image and their SHA's (should be `quay.io` or `registry.redhat.io` images IMHO).

Reply:
Yeah, there is something called renovators, which I need to check.